api/tests/routes/adage/v1/confirm_prebooking_test.py

```diff
@@ -25,7 +25,7 @@
 
 @freeze_time("2021-10-15 09:00:00")
 class Returns200Test:
-    def test_confirm_collective_prebooking(self, client, caplog) -> None:  # type: ignore [no-untyped-def]
+    def test_confirm_collective_prebooking(self, client, caplog) -> None:
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🤓 Détail: ** ‏dans la CI, mypy ignore les tests !
Donc ce n'est pas nécessaire, mais ça me va tout à fait :+1:

Reply:
C'est justement parce que mypy ignore les tests que j'ai enlevé les `type: ignore` dedans, ça évite de les ajouter quand je les recherche dans le projet :)